genkit-tools/common/src/server/router.ts

```diff
@@ -191,6 +191,15 @@ export const TOOLS_SERVER_ROUTER = (manager: RuntimeManager) =>
         return evalRun;
       }),
 
+    /** Deletes a single eval run by ID */
+    deleteEvalRun: loggedProcedure
+      .input(apis.DeleteEvalRunRequestSchema)
+      .mutation(async ({ input }) => {
+        const parts = input.name.split('/');
+        const evalRunId = parts[1];
```

Comment:
Let's also verify parts[0] and throw incorrect format errors?

Please also add a test for this.

Reply:
I don't think this is super crucial. This is not user facing directly so the benefit of having extra validations is very little. I'll leave this as it is to match other routes here.